python/test/integration/test_histogram.py

```diff
@@ -32,20 +32,19 @@ def test_count_by_categories_float():
     load = dp.t.make_split_dataframe(",", ["A", "B"]) >> dp.t.make_select_column(
         "A", TOA=str
     )
-    
-    noisy_float_histogram = (
-        load
-        >> dp.t.then_count_by_categories(cats, MO=dp.L1Distance[float], TOA=float)
-        >> dp.m.then_base_laplace(scale=1.0)
+
+    t_hist = load >> dp.t.then_count_by_categories(
+        cats, MO=dp.L1Distance[float], TOA=float
     )
-    print(noisy_float_histogram(data))
+    assert t_hist(data) == [5.0, 20.0, 10.0, 5.0]
+    # ensure that chaining works as expected
+    (t_hist >> dp.m.then_laplace(1.0))(data)
 
-    noisy_float_histogram = (
-        load
-        >> dp.t.then_count_by_categories(cats, MO=dp.L2Distance[float], TOA=float)
-        >> dp.m.then_base_gaussian(scale=1.0)
+    t_hist = load >> dp.t.then_count_by_categories(
+        cats, MO=dp.L2Distance[float], TOA=float
     )
-    print(noisy_float_histogram(data))
+    assert t_hist(data) == [5.0, 20.0, 10.0, 5.0]
+    (t_hist >> dp.m.then_gaussian(1.0))(data)
```

Comment:
At some point, remind me why we can't set the noise to zero or something very very small and make a stronger assertion.

Reply:
I should have just done that here.